src/jupyter_contrib_nbextensions/nbextensions/export_embedded/readme.md

```diff
@@ -0,0 +1,7 @@
+Export Embedded HTML
```

Comment:
Could we alter this to something like "Export HTML with images embedded" to clarify what's going on?